app/views/banner/TopBannerView.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+import React from 'react'
+import {
+	View,
+	TouchableHighlight,
+  Image,
+	Text,
+  Modal
+} from 'react-native';
+
+import BannerView from './BannerView';
+
+var css =             require('../../styles/css');
+var general = 			  require('../../util/general');
+var AppSettings = 		require('../../AppSettings');
+
+// Manages the top 'hero' banner & decides what if anything to show
+// Currently decides based on static dates but an API would be great for breaking info
+export default class TopBannerView extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
@jpknoll yea that was my thought.  Eventually I'd like this component to get its data externally and then load the results into state.